example/src/screens/calendars.js

```diff
@@ -27,6 +27,22 @@ const CalendarsScreen = () => {
     return disabledDates;
   };
 
+  const dayUnderView = () => {
+		return (
+			<View>
+				<Text>This is day under view for custom</Text>
+			</View>
+		);
+	};
+
+	const dayUnderView2 = () => {
+		return (
+			<View>
+				<Text>Or show any icon as here</Text>
+			</View>
+		);
+	};
+
```

Comment:
Could you reformat this file with spaces instead of tabs?

Reply:
The space is same with before code, not commit as above code.